### ISSUE TITLE
NuGet Package, Part II

### ISSUE THREAD

**Issue by kevingy** [seanmcelroy/nntp-client#3]
The current NuGet package targets .NETCoreApp 2.1. My app is .NETCore 3.1. I'm getting warnings like:

![image](https://user-images.githubusercontent.com/411556/91240583-0782f900-e708-11ea-8497-d065ee5da599.png)

Perhaps the library should target .NETStandard for better compatibility with more targets?

**Comment by seanmcelroy** [comment]
Good point.  I'll have to refresh my understanding of CoreApp vs Core vs Standard as it relates to Nuget best practices.

**Comment by kevingy** [comment]
In a perfect world, all that is required is a change to the project file to change the target. I haven't looked at the project yet to know if more is required.
